recipe/meta.yaml

```diff
@@ -1,32 +1,39 @@
-{% set version = "15.0.7" %}
+{% set version = "16.0.0" %}
 
 package:
   name: lld
   version: {{ version }}
 
 source:
   url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: 8b5fcb24b4128cf04df1b0b9410ce8b1a729cb3c544e6da885d234280dedeac6
+  sha256: 9a56d906a2c81f16f06efc493a646d497c53c2f4f28f0cb1f3c8da7f74350254
   patches:
     - patches/0001-point-header-inclusion-in-lld-MachO-CMakeLists.txt-t.patch
 
 build:
   number: 0
-  skip: true  # [win and vc<14]
 
 requirements:
   build:
-    - cmake >=3.4.3
-    - ninja    # [win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make     # [not win]
-    - llvmdev =={{ version }}    # [build_platform != target_platform]
+    - cmake
+    - ninja
+    - llvmdev =={{ version }}              # [build_platform != target_platform]
   host:
+    - libcxx {{ cxx_compiler_version }}    # [osx]
     - llvmdev =={{ version }}
     - llvm
-    - libxml2  # [win]
+    # note: these libs may show up as unused in the link check,
+    # but not all the artefacts can be inspected by conda (e.g.
+    # static libs); however, they are present in the host env
+    # as run-deps of llvm and so they will be picked up for the
+    # build by the default settings.
```

Comment:
Can you update this comment as we did in clangdev-feedstock?

Reply:
In https://github.com/conda-forge/clangdev-feedstock/pull/206, we don't have a comment on the top-level host env anymore, so I'm not 100% sure which one you'd like me to mirror (`# Use the same requirements as the top-level requirements` quite clearly makes no sense).

I updated the comment anyway and ignored a run-export that showed up unused on unix (only).